resources/lib/jellyfin/core/api.py

```diff
@@ -64,12 +61,7 @@ def sessions(self, handler="", action="GET", params=None, json=None):
         else:
             return self._get("Sessions%s" % handler, params)
 
-    def users(self, handler="", action="GET", params=None, json=None):
-        if action == "POST":
-            return self._post("Users/{UserId}%s" % handler, json, params)
-        elif action == "DELETE":
-            return self._delete("Users/{UserId}%s" % handler, params)
-        else:
+    def users(self, handler="", action="GET", params=None, json=None): ## Remove action & JSON, maybe others
```

Comment:
These changes are potentially breaking (`POST` and `DELETE` are in use), and unrelated to wake on lan.